code/datums/repositories/crew/vital.dm

```diff
@@ -24,7 +24,7 @@
 				if(PULSE_THREADY)
 					crew_data["alert"] = TRUE
 					crew_data["pulse_span"] = "bad"
-		else if(BP_IS_ROBOTIC(O))
+		else if(O)
```

Comment:
No need to check for O twice, you can put one if(O) and these 2 tabbed under it.